storage/onpremise/mongodb/outputs.tf

```
@@ -43,3 +43,37 @@ output "unused_variables" {
     "validity_period_hours" = var.validity_period_hours
   }
 }
+
+#new Outputs 
+output "env" {
+  description = "Elements to be set as environment variables"
+  value = ({
+    "Components__TableStorage"  = var.adapter_class_name
+    "MongoDB__Host"             = local.mongodb_dns
+    "MongoDB__Port"             = "27017"
+    "MongoDB__Tls"              = var.tls
+    "MongoDB__ReplicaSet"       = var.replicaset
+    "MongoDB__DatabaseName"     = var.mongodb.databases_names[0]
```

Not dependent of you but I think we should hardcode this one to `database`
```suggestion
    "MongoDB__DatabaseName"     = "database"
```

---

I thought I had deleted this option but no

---

is the `databases_names` exposed ? if that is the case, better not to hardcode

---

It is not exposed. It is hardcoded in ArmoniK configuration.